src/modules/whats-new/components/whats-new-modal.tsx

```diff
@@ -18,70 +18,72 @@ interface WhatsNewModalProps {
 	onClose: () => void;
 }
 
-const WHATS_NEW_PAGES: WhatsNewPage[] = [
-	{
-		image: versionSwitchIllustration, // TODO: Add correct illustration for intro card
-		title: __( 'What is new in Studio?' ),
-		description: __(
-			'Discover the latest updates in Studio! Explore exciting new features designed to enhance your experience.'
-		),
-	},
-	{
-		image: previewSitesIllustration, // TODO: Add correct illustration
-		title: __( 'Choose a custom domain with HTTPS support' ),
-		description: __(
-			'Easily identify your local Studio sites with custom domain names. Personalize and organize your workflow!'
-		),
-		learnMoreUrl: 'https://wordpress.com/blog/2025/03/31/studio-custom-domains-https/',
-	},
-	{
-		image: versionSwitchIllustration,
-		title: __( 'Select WordPress and PHP versions in Studio' ),
-		description: __(
-			'Select your preferred WordPress and PHP versions for existing sites or when creating a new one.'
-		),
-		learnMoreUrl: 'https://wordpress.com/blog/2025/03/17/studio-wordpress-php-versions/',
-	},
-	{
-		image: previewSitesIllustration,
-		title: __( 'Share your work easily with Preview sites' ),
-		description: __(
-			'Quickly generate a publicly accessible URL that you can share with clients and colleagues.'
-		),
-		learnMoreUrl: 'https://wordpress.com/blog/2025/02/24/studio-preview-sites/',
-	},
-];
-
 const PageContent = ( {
 	title,
 	description,
 	learnMoreUrl,
 	isIntroPage = false,
-}: Omit< WhatsNewPage, 'image' > & { isIntroPage?: boolean } ) => (
-	<div className="px-8 pt-4 pb-2 flex flex-col h-full">
-		<h2 className="text-xl mb-4 text-gray-900 line-clamp-2">{ title }</h2>
-		<p
-			className={ cx(
-				'text-gray-900 text-m leading-s',
-				isIntroPage ? 'line-clamp-5' : 'line-clamp-3'
-			) }
-		>
-			{ description }
-		</p>
-		<div className="mt-2 mb-4">
-			{ learnMoreUrl && (
-				<button
-					onClick={ () => getIpcApi().openURL( learnMoreUrl ) }
-					className="text-a8c-blueberry text-m leading-s cursor-pointer"
-				>
-					{ __( 'Learn more' ) }
-				</button>
-			) }
+}: Omit< WhatsNewPage, 'image' > & { isIntroPage?: boolean } ) => {
+	return (
```

Comment:
What is the reasoning behind this change?

Reply:
Good catch. Thanks! That was an unintentional leftover from AI. I have put the original back.